src/test/java/org/folio/rest/api/FixedDueDateApiTest.java

```diff
@@ -231,7 +232,7 @@ public void canCreateSchedule()
       ResponseHandler.json(updateCompleted4));
     JsonResponse updateCompleted4Response = updateCompleted4.get(5, TimeUnit.SECONDS);
     assertThat(String.format("Failed to create due date: %s", fixDueDate5.encodePrettily()),
-      updateCompleted4Response.getStatusCode(), is(HttpURLConnection.HTTP_BAD_REQUEST));*/
+      updateCompleted4Response.getStatusCode(), is(HttpStatus.HTTP_UNPROCESSABLE_ENTITY.toInt()));*/
```

Comment:
Maybe remove it at all, since it is commented.

Reply:
Yeah, I think we should remove this commented out code